### PR TITLE
#182 Introduced `range-of-ints` object

### PR DESCRIPTION
**Pull request by maxonfjvipon**
Closes: #182

---

## PR-Codex overview
This PR adds a range-of-ints object with a test suite and a todo comment for simplifying the object. 

### Detailed summary
- Added `range-of-ints` object with start and end integer arguments
- Added a test suite for `range-of-ints` object
- Added a todo comment for simplifying the `rec-filtered` object inside `filteredi`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by maxonfjvipon** [comment]
@yegor256 please have a look

File: src/test/eo/org/eolang/collections/range-of-ints-tests.eo [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
++alias org.eolang.collections.list
++alias org.eolang.collections.range-of-ints
++alias org.eolang.hamcrest.assert-that
++home https://github.com/objectionary/eo-collections
++package org.eolang.collections
++tests
++version 0.0.0
+
+[] > simple-range-of-ints
```
Comment (yegor256) [inline src/test/eo/org/eolang/collections/range-of-ints-tests.eo:31]: @maxonfjvipon maybe it would useful to add a test for two empty ranges: `range-of-ints 10 10` and `range-of-ints 5 2`

**Comment by yegor256** [comment]
@maxonfjvipon see above

File: src/test/eo/org/eolang/collections/range-of-ints-tests.eo (hunk 1 above)
Comment (maxonfjvipon) [reply src/test/eo/org/eolang/collections/range-of-ints-tests.eo:31]: @yegor256 Done

**Comment by yegor256** [comment]
@rultor merge
